proposed/0000-Farm-attempt.md

```diff
@@ -0,0 +1,56 @@
+- Feature Name:Farm attempt 
+- Type new feature
+- Related components maidsafe_vault
+- Start Date: 24-06-2015 
+- RFC PR:
+- Issue number: 
+
+# Summary
+
+This proposal outlines an initial design to test the efficiency and security of farming attempts.  This involves making use of the Farming Rate (FR) which is the rate at which farming attempts are made.  It is assumed the farming rate will include payments to care development, content producers and most significantly farmers (or providers of the network resource). 
+
+# Motivation
+
+To ensure network stability and resource availability a reward mechanism is required to ensure the use of resource is balanced with the supply of resource. 
```

Comment:
I would change
"To ensure network stability and resource availability" 
to 
"As an incentive to improve network stability and resource availability"